xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -1441,7 +1441,7 @@ This method is a no-op and added only for consistency: the `value` is returned u
         <Parameter Name="value" Type="System.UInt32" />
       </Parameters>
       <Docs>
-        <param name="destination">The = span of bytes where the value is to be written as big endian.</param>
```

Comment:
I am curious to know what caused this issue initially. Bad merge conflict or issue with tool that ports the text?